.github/workflows/JuliaFormatter.yml

```diff
@@ -36,8 +36,8 @@ jobs:
       if: steps.filter.outputs.julia_file_change == 'true'
       run: |
         julia --project=.dev .dev/climaformat.jl .
-          
+
     - name: Check formatting diff
       if: steps.filter.outputs.julia_file_change == 'true'
       run: |
-        git diff --color=always --exit-code
\ No newline at end of file
+        git diff --color=always --exit-code
```

Comment:
meant to comment here - changed on purpose?

Reply:
oops, no. I'll change it back